.github/workflows/pythonapp.yml

```diff
@@ -20,7 +20,7 @@ jobs:
             - EXTRAPKGS:
 
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
```

Comment:
I'm so glad this immediately did the trick

Reply:
I need to get up to speed on Github actions!